packages/file-search/src/node/file-search-service-impl.spec.ts

```diff
@@ -54,7 +54,7 @@ describe('search-service', function (): void {
         expect(testFile).to.be.not.undefined;
     });
 
-    it.skip('shall respect nested .gitignore', async () => {
+    it('shall respect nested .gitignore', async () => {
```

Comment:
Please revert. It does not work.

Reply:
Reverted.

I had thought this test was fixed because it worked on both my linux and windows machines, but it's still failing on appveyor.